docs/source/developer/test_suite.rst

```diff
@@ -8,8 +8,79 @@ Test Suite
 Running the Test Suite
 ==============================================================================
 
-``./csu dev test`` will run the entire test suite. For running a specific tests, viewing code coverage, and more testing commands, see :doc:`../getting_started/helper_commands`.
+``./csfg test`` will run the entire test suite. For running a specific tests, viewing code coverage, and more testing commands, see :doc:`../getting_started/helper_commands`.
```

Comment:
`ERROR: Unknown command!`
According to `./csfg` the commands are:
test_suite
test_specific
test_coverage
test_backwards